docs/root/intro/deployment_types/double_proxy.rst

```diff
@@ -22,5 +22,4 @@ Configuration template
 ^^^^^^^^^^^^^^^^^^^^^^
 
 The source distribution includes an example double proxy configuration that is very similar to
-the version that Lyft runs in production. See :ref:`here <install_ref_configs>` for more
-information.
+the version that Lyft runs in production.
```

Comment:
This isn't actually true anymore. Can you remove this statement here and elsewhere as long as you are in here? I would just make this more generic in terms of showing a rudimentary templated configuration system or something like that.